packages/dds/tree/src/test/simple-tree/api/discrepancies.spec.ts

```diff
@@ -0,0 +1,561 @@
+/*!
```

Comment:
This file was mostly moved. Annoyingly the github diff didn't figure that out. I left some comments on the removed version of this file about what was changed.

Reply:
Note that the actual tests for schemaCompatabilityTester check that the user facing implications of  getAllowedContentDiscrepancies are correct, so the exact details of these tests don't seem super important.

Most of the changes to these tests were removing generation of discrepancies which schemaCompatabilityTester used to ignore and thus we no longer generate, and inlining of the node's field discrepancies.